failover-slots/README.md

```diff
@@ -0,0 +1,24 @@
+# Failover Slots with Postgres 17
+
+This example shows the usage of Postgres 17 failover replication slots with Postgres SQL-based interface to logical replication as well as with Decodable.
+It accompanies the blog post <TODO>.
```

Comment:
tiny nit: is it better to just put decodable.co/blog and then change it once the URL is live? that way if you forget / someone gets to the readme before you update it, they don't have a dead link.